### PR TITLE
sort resources by alphabet

### PR DESCRIPTION
**Pull request by mugnimaestra**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,66 +4,20 @@
...
-- [Smashing Magazine](https://www.smashingmagazine.com)
-- [Front End Masters Handbook](https://frontendmasters.com/books/front-end-handbook/2018/)
-- [Javascript Weekly](https://javascriptweekly.com/)
-- [Frontend Focus](https://frontendfoc.us/)
-- [Rand Fishkin at Moz](https://moz.com/community/users/63)
+# Documentation for the web
+  - [All W3C Standards and Drafts](https://www.w3.org/TR/)
+  - [Google's Web Fundamentals](https://developers.google.com/web/fundamentals/)
+  - [MDN Web Docs](https://developer.mozilla.org/en-US/)
+    - [all HTML elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element)
```
Comment (osmarpetry) [inline README.md:17]: Why this not necessary blank space?

File: README.md [hunk 2]
```diff
@@ -160,10 +80,90 @@
...
+- [Hacker News](https://news.ycombinator.com/)
+- [Javascript Weekly](https://javascriptweekly.com/)
+- [Rand Fishkin at Moz](https://moz.com/community/users/63)
+- [Smashing Magazine](https://www.smashingmagazine.com)
+
+# Study Materials
+  - [What is Code](https://www.bloomberg.com/graphics/2015-paul-ford-what-is-code/)
+
+  - ## Javascript
+    - [JavaScript for Cats](https://noblemule.gitbooks.io/javascript-for-cats/content/)
```
Comment (osmarpetry) [inline README.md:128]: More not necessary blank space again

File: README.md [hunk 3]
```diff
@@ -160,10 +80,90 @@
...
+- [Free Sales force training for Vets](https://veterans.force.com/NewVetForceHome)
+- [Linkedin for Vets](https://socialimpact.linkedin.com/programs/veterans/premiumform)
+
+# Web Design Inspiration
+- [Brutalist Design](http://brutalistwebsites.com/)
+- [Collect UI](https://medium.com/@CollectUI)
+  
+# Web Security
+- ### Guides
+  - [Awesome Reference Guide from Mozzilla's Infosec Team](https://infosec.mozilla.org/guidelines/web_security.html)
```
Comment (osmarpetry) [inline README.md:159]: Also blank space here and the following lines

**Review by osmarpetry — CHANGES_REQUESTED** [review]
Excellent job

File: README.md (hunk 1 above)
Comment (mugnimaestra) [reply README.md:17]: I thought because it is part of MDN Web Docs, or should I just make it same level with other list?

File: README.md (hunk 2 above)
Comment (mugnimaestra) [reply README.md:128]: noted

File: README.md (hunk 3 above)
Comment (mugnimaestra) [reply README.md:159]: I'm sorry I don't quite get what you mean about this one, where I need to add blank space and the following lines ?

File: README.md (hunk 1 above)
Comment (axecopfire) [reply README.md:17]: I think nesting is a good idea. I don't think you need an extra line after.

**Review by axecopfire — APPROVED** [review]

**Pull request merged**